RealifeTech-CoreSDK/APIV3Utilities/Tests/AuthorisationWorkerTests.swift

```diff
@@ -66,24 +66,69 @@ final class AuthorisationWorkerTests: XCTestCase {
             }).disposed(by: disposeBag)
         MockOAuthRepository.accessTokenSource.onNext(initialToken)
     }
+
+    func test_refreshAccessToken_hasRefreshTokenAndAccessTokenIsExpired_tokenIsStored() {
+        storeSpy.accessTokenValid = false
+        storeSpy.accessToken = testTokenString
+        storeSpy.refreshToken = testTokenString
+        let testOAuthToken = OAuthToken(
+            accessToken: testTokenString,
+            refreshToken: testTokenString,
+            expiresIn: testExpirySeconds,
+            tokenType: nil,
+            scope: nil)
+        let expectation = XCTestExpectation(description: "OAuth observable did emit value")
+        emitRefreshAccessToken(initialToken: testOAuthToken) { [self] token in
+            XCTAssertEqual(storeSpy.savedCredentials?.0, testTokenString)
```

Comment:
May consider adding XCTUnwrap here:
```
let credentials = try XCTUnwrap(storeSpy.savedCredentials)
```

Reply:
`XCTAssertEqaul` is better here in my opinion as it won't be equal if it's nil

Reply:
I was thinking of adding it in addition. However I didn't realise that XCTAssertEqual showed a warning message that included nil. `XCTAssertEqual failed: ("Optional("String")") is not equal to ("nil")`